.github/workflows/build-on-master.yml

```
@@ -0,0 +1,24 @@
+name: Build on master
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [1.7, 1.8, 11, 14]
```

```suggestion
        java: [1.8, 11, 14]
```

Removed `JDK 1.7` as `gradlew` wrapper requires JDK >= 1.8